boards/adafruit/feather_esp32/adafruit_feather_esp32_appcpu.yaml

```diff
@@ -0,0 +1,27 @@
+identifier: adafruit_feather_esp32/esp32/appcpu
+name: Adafruit ESP32 Feather
```

Comment:
Would you mind adding `APPCPU` and `PROCPU` accordingly in both yaml files?
`name: Adafruit ESP32 Feather APPCPU`

Reply:
Done, added to the dts files too